### DISCUSSION TITLE
Design Lab 平台最近一段时间不好用

### DISCUSSION THREAD

**Discussion by liucangsong** [arco-design/arco-design#2885] [General]
Arco Design 一直是非常优秀的组件平台，特别是Design Lab，非常棒的体验。 但是Design Lab最近几个月来一直问题多多。真希望各位神维护一下这个完美的平台。

**Comment by yinkaihui** [comment]
在做了。。 在做了。。 😭

**Reply by yinkaihui** [reply]
更新了一版，可以先试试，应该会好一点

**Reply by liucangsong** [reply]
完美的体验又回来了！感谢，感谢。

**Reply by liucangsong** [reply]
好像又出问题了。 已经看不到自己创建的样式了。

**Comment by fx89757** [comment]
大哥些，访问不了了。创建不了，查看自己创建的也不展示了
